crates/cli-support/src/js/js2rust.rs

```diff
@@ -138,15 +151,22 @@ impl<'a, 'b> Js2Rust<'a, 'b> {
         self
     }
 
-    fn abi_arg(&mut self) -> String {
-        let s = format!("arg{}", self.arg_idx);
+    fn abi_arg(&mut self, opt_arg_name: Option<String>) -> String {
```

Comment:
Could this perhaps be `Option<&str>` to avoid some extra clones as necessary in a few places?

Reply:
I wish! syn's Ident doesn't expose `&str` and only implements `ToString`.